Stages.md

```diff
@@ -0,0 +1,131 @@
+
+Stage signals are also useful to the broader community, helping developers monitor the proposals that are moving through the various stages.  By explicitly signaling a proposal's progress, including implementer involvement, the community has a better idea of what is going on in the WHATWG.
+
+This tool is generally used for medium-to-large new features; it's not expected that explicit stage progression will be used for every feature.
+
+# Details
+
+This document formalizes some signal checkpoints for new features in existing WHATWG Living Standards. These checkpoints are modeled loosely on the [TC39 process](https://tc39.es/process-document/), which uses the concept of _stages_. Each subsequent stage implies a larger degree of consensus from the community, willingness to engage and/or implement and ship the feature in browser engines, and signals progress to web developers.  These checkpoints are not required for all features; this is simply a way to ensure that attention is appropriately drawn to incubations that have grown community support behind them.
+
+## Terminology
+
+* <dfn>Stage</dfn>: each successive stage is closer to a landed spec and interoperable implementation.  These stages are tracked on the issues in the relevant WHATWG GitHub repository, using labels ("stage 0", "stage 1", etc.)
+* <dfn>Browser engine</dfn>: an independent implementation of the web platform (e.g. Chromium, WebKit and Gecko at present).
+* <dfn>Implementation interest</dfn>: a direct signal from a browser engine that a feature is on the roadmap to implement, as defined in [working mode](https://whatwg.org/working-mode#additions)].
+* <dfn>Browser engine representatives</dfn>: persons delegated by a browser engine to indicate implementation interest.
+* <dfn>Proposal champion</dfn>: person/organization that proposed and will advance a feature through the stages, typically including writing the specification, researching use cases, and prototyping.
+
+## Process
+
+* Any proposal effectively starts at Stage 0 without any approvals, by a community member filing a new issue on a relevant WHATWG spec.
+* In order to drive a proposal forward, it is expected that a community member will step up to champion the proposal, through answering questions asynchronously, providing feedback when requested, and consideration of important open questions at triage meetings.
+* Stage level is tracked with labels on issues in the relevant specifications — “stage 0”, “stage 1”, etc.
+* Stage labels (other than stage 0) should only be added by browser engine representatives, or by Editors on the relevant specification, after showing the necessary support for a level.
```

Comment:
Suggested addition:

* This process is subject to and governed by [WHATWG Policies](https://whatwg.org/policies), including the Intellectual Property Rights Policy.

Reply:
Addressed in https://github.com/whatwg/sg/pull/227/commits/b8175528d6da72c791fb996a228ad13fde1f25e2.